pkg/npm/api/hood/types.ts

```diff
@@ -0,0 +1,154 @@
+
+/**
+ * A pending commit, awaiting a future kelvin version
+ */
+interface Woof {
+  aeon: number;
+  weft: Weft;
+}
+
+interface Rein {
```

Comment:
FYI I recently added a `liv=?` field to `$rein`, to model the "whole desk is suspended" state.